src/CreateFlagCommand.php

```diff
@@ -17,12 +16,22 @@ public function __construct(private CreateFlag $createFlag)
         parent::__construct();
     }
 
-    public function handle()
+    public function handle(): void
     {
+        $name = $this->option('name');
+        $description = $this->option('description');
+        $tags = $this->option('tags');
+
+        if (!is_string($name) || !is_string($description) || !is_array($tags)) {
+            $this->error('Please provide a valid name, description and tags');
+
+            return;
```

Comment:
**suggestion:** a good practice is to return a [proper exit code](https://github.com/symfony/console/blob/5.3/Command/Command.php#L34-L36)
```suggestion
            return Command::FAILURE;
```